content/language/hoon/reference/stdlib/2k.md

```diff
@@ -322,7 +322,7 @@ A cell of the last element in `a` along with the rest of queue `a`.
 ```
 > =s (~(gas to *(tree @)) `(list @)`~[1 2 3])
 > ~(get to s)
-[n=2 l={3} r={1}]
+[p=1 q={3 2}]
 
 > ~(get to ~)
 ! exit
```

Comment:
`dojo: hoon expression failed`

Reply:
@ripperi wdym? I copy the lines to Dojo and get the same output as in my PR

![image](https://github.com/urbit/docs.urbit.org/assets/12774450/9ee29506-62b9-45db-b468-56e5fdcdcf76)

Reply:
The failure case `~(get to ~)` output seems to be outdated as well. Might as well update that in the process.

Reply:
sure, I made a commit